catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +267,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Remove whitespace and literal newlines
+     * @param metadata
+     * @return
+     */
+    private String sanitize(final String metadata) {
+        return metadata.replaceAll("[\\s|\\n|\\r|\\t]+", "");
```

Comment:
we don't want to replace _all_ whitespace, just excess whitespace. i think the `\\s` is overzealous